modules/nw-openshift-sdn-modes.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_sdn/about-openshift-sdn.adoc
+
+[id="nw-openshift-sdn-modes_{context}"]
+= OpenShift SDN network isolation modes
+
+ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
+OpenShift SDN provides three SDN modes for configuring the pod network:
+
+* The _network policy_ mode allows project administrators to configure their own
```

Comment:
I'm rolling deep in article constructs here...for readability, dropping "the" from each mode might emphasize the modes more:
- _Network policy_ mode allows...
- _Multitenant_ mode provides...
- _Subnet_ mode provides...